### ISSUE TITLE
tests failing: ModuleNotFoundError: No module named 'tensorflow_probability'

### ISSUE THREAD

**Issue by cwmeijer** [dianna-ai/dianna#270]
I checked a couple of the failing tests. All of them only had problems finding a package:

```python
import tensorflow as tf
>   from tensorflow_probability import distributions as tfd
E   ModuleNotFoundError: No module named 'tensorflow_probability'
```
Sounds like something that broke in that package last week? I haven't looked into it further.

**Comment by cwmeijer** [comment]
It seems we are just missing this dependency. Maybe one of our dependencies has a dependency on this package but dropped it lately while still using it? I'll check it out.

**Comment by cwmeijer** [comment]
I fixed the version of onnx_tf to 1.9 in PR #273 which seems to fix the issue. However, the build for onnx-tensorflow is completing just fine (https://github.com/onnx/onnx-tensorflow). I even tried importing onnx_tf on google collab where it also succeeded.

Quite puzzling why the latest version onnx_tf 1.10 gives me the above import error on:
- our github actions
- my local Windows environments
- my WSL ubuntu environments
